C-Plus-Plus/cp/aggressive_cows.cpp

```diff
@@ -0,0 +1,80 @@
+/*Farmer John has built a new long barn, with N (2 <= N <= 100,000) stalls.
+The stalls are located along a straight line at positions x1,...,xN (0 <= xi <= 1,000,000,000).
+His C (2 <= C <= N) cows don't like this barn layout and become aggressive towards each other once put into a stall. 
+To prevent the cows from hurting each other, FJ wants to assign the cows to the stalls, 
+such that the minimum distance between any two of them is as large as possible. 
+What is the largest minimum distance?*/
+
+#include <bits/stdc++.h>
+using namespace std;
+bool canplacecows(int stalls[], int n, int c, int min_sep)
+{
+	int last_cow = stalls[0];
+	int cnt = 1;
+	for (int i = 1; i < n; i++)
+	{
+		/* Checking if the placing of cows is greater
+		ie the next cows placed aren't next to each other */
+		if (stalls[i] - last_cow >= min_sep)
+		{
+			last_cow = stalls[i];
+			cnt++;
+			if (cnt == c)
+			{
+				return true;
+			}
+		}
+	}
+
+	return false;
+}
+
+int main()
+{
+	int stalls[100];
+	int n;
+	int cows;
```

Comment:
Shift these variable declaration to a single line i.e `int stalls[100], n, cows;`